### PR TITLE
Attempting to fix rendering problems on jujucharms.com (again)!

### PR DESCRIPTION
**Pull request by mbruzek**
I also changed some paragraphs to be less than 80 columns.
I added some words on Ingress and what it is.
Fixed some other spots.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -33,60 +33,53 @@ developers, use the smaller
...
-This will deploy the Canonical Kubernetes offering with default constraints.
-This is useful for lab environments, however for real-world use you should
-provide high CPU and memory instances to kubernetes-worker.
+This will deploy the Canonical Distribution of Kubernetes offering with default
+constraints. This is useful for lab environments, however for real-world use
+you should provide higher CPU and memory instances to kubernetes-worker units.
 
-You can do this by editing the [bundle](https://github.com/juju-solutions/bundle-canonical-kubernetes)
-to fit your needs, it is commented for your convenience.
+You can increase the constraints by editing the
```
Comment (chuckbutler) [inline README.md]: Did we leave in commented constraints? I think that needs to be re-introduced. Its not a blocker, but we're currently telling a fib here....

File: README.md [hunk 2]
```diff
@@ -33,60 +33,53 @@ developers, use the smaller
...
-This means those charms are accessible through the public addresses.
+This bundle exposes the kubeapi-load-balancer and kibana applications by 
+default. This means those charms are accessible through the public addresses.
 
 If you would like to remove external access, unexpose the applications:
 
 ```
-
 juju unexpose kibana
 juju unexpose kubernetes
```
Comment (chuckbutler) [inline README.md]: There is no 'kubernetes' application now, its either kubernetes-master or kubernetes-worker

File: README.md [hunk 3]
```diff
@@ -189,36 +174,37 @@ about configuring kubectl with the
...
+This action performs the following steps:
 
-It creates a deployment titled 'microbots' comprised of 5 replicas defined
+- It creates a deployment titled 'microbots' comprised of 5 replicas defined
 during the run of the action. It also creates a service named 'microbots'
 which binds an 'endpoint', using all 5 of the 'microbots' pods.
 
-Finally, it will create an ingress resource, which points at a [xip.io](https://xip.io)
-domain to aid in simulating having a proper DNS
+- Finally, it will create an ingress resource, which points at a
```
Comment (chuckbutler) [inline README.md]: I realize I did this initially, but cold you add a bit about how to clean it up?

```
juju run-action kubernetes-worker/0 microbot delete=true
```

File: README.md (hunk 1 above)
Comment (mbruzek) [inline README.md]: We do have constraints commented in the bundle.

File: README.md (hunk 2 above)
Comment (mbruzek) [inline README.md]: kubeapi-load-balancer actually but yeah I missed this one.

**Comment by mbruzek** [comment]
I addressed your comments. Please re-review.

**Pull request merged**